website/components/remote-plugin-docs/server.js

```diff
@@ -21,10 +23,12 @@ async function generateStaticProps(
   navDataFile,
   localContentDir,
   params,
-  { productName, remotePluginsFile, additionalComponents } = {}
+  product,
+  { remotePluginsFile, additionalComponents, mainBranch = 'main' } = {}
```

Comment:
Additional args here are because we're constructing the `Edit this page` URL here in `getStaticProps` rather than within the `docs-page` component as we were previously doing on the pre-release.